drivers/serial/uart_bt_auto_start_bt.c

```diff
@@ -0,0 +1,43 @@
+/*
```

Comment:
This has technically nothing to do with uart, it's pretty much like a systemd service file for BLE.
But it's hard to find a place where it would belong, yeah.

Maybe in ~/repos/zephyrproject/zephyr/subsys/bluetooth/lib` ? or next to the nus-service.